index.html

```diff
@@ -289,6 +289,26 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact_form_container">
+        <section class="contact_form_section">
+          <h2 class="contact_form_header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</h2>
+          <div class="form_container">
+            <form class="contact form" action="">
+              <div class="name">
+                <input type="text" name="Name" id="first_name" placeholder="First name">
+                <input type="text" name="Surname" id="last_name" placeholder="Last name">
+              </div>
+              <div class="email">
+                <input type="text" name="Email" id="email_address" placeholder="Email address">
+              </div>
+              <div class="info">
+                <textarea name="Info" id="information" cols="30" rows="10" placeholder="Write your message here" maxlength="500"></textarea>
+              </div>
+              <input type="submit" class="submit_button">
```

Comment:
- [ ] **[REQUIRED CHANGES]** This button (submit input, in your case) should contain the text: 'Get in Touch', in order for the user to know what they should expect from the button. You can use the `value` property to do it.